RecoHI/HiTracking/python/HITrackingRegionProducer_cfi.py

```diff
@@ -88,3 +88,15 @@
 HiTrackingRegionFactoryFromSTAMuonsBlock.MuonTrackingRegionBuilder.vertexCollection = cms.InputTag("hiSelectedVertex")
```

Comment:
According to `git grep` `hiRegitMuonLowPtTripletStep` is not currently used anywhere, hence I didn't migrate it (my strategy for HI was to be almost-minimal to keep `140.53` working). Therefore before migrating `hiRegitMuonLowPtTripletStep` I'd like to get a confirmation that it will be used in the future (otherwise I'd remove it).